document-ai/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "Apache-2.0",
   "author": "Google LLC",
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=14.0.0"
```

Comment:
I think we may want to avoid having samples that require Node 14, since our libraries are still supporting Node 12. Our best option may be avoiding updates to some of these dependencies, unless there's a forcing function (such as a security vulnerability).

I'm all for dropping Node 12 for client libraries, but we need a plan for how we announce this to customers.